index.rst

```diff
@@ -62,19 +58,24 @@ Who to Contact for What
    :header-rows: 1
 
    * - Situation
-     - Contact
+     - Primary Contact
    * - Gerrit/Jenkins Outage
-     - Sysadmin Team
+     - Sysadmin
    * - Gerrit/Jenkins Issues (not an outage)
-     - Gluster CI Engineer
+     - CI Engineer
    * - Access to machines
-     - Sysadmin Team
+     - Sysadmin
    * - Access to build nodes (previously called slaves)
-     - Gluster CI Engineer
+     - CI Engineer
    * - Infra-related build failures
-     - Gluster CI Engineer
+     - CI Engineer
    * - Infra Security Issues
-     - Sysadmin and Gluster CI Engineer
+     - Sysadmin and CI Engineer
+
+If the primary contact is not present (days off, holidays, sick), another member of the infrastructure team can 
+take a look and solve issues. Everybody in the infra team has the same access, and if anything is missing for a member,
```

Comment:
Define issues specifically as Emergencies as defined in out documentation. We do not expect either roles to fill in for day-to-day activities for each other due to lack of contextual knowledge, among other things.